modules/telestion-application/src/main/java/de/wuespace/telestion/application/launcher/Launcher.java

```diff
@@ -0,0 +1,83 @@
+package de.wuespace.telestion.application.launcher;
+
+import io.vertx.core.Future;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.lang.Thread.sleep;
+
+/**
+ * A {@link Launcher} is the first part of the launch process in Telestion.
+ * <p>
+ * It starts other processes like {@link de.wuespace.telestion.application.loader.Loader Loaders}
+ * or {@link io.vertx.core.Verticle verticles}, configures {@link io.vertx.core.Vertx} instances or runs external code.
+ * <p>
+ * A launcher has a {@link #start() start} and {@link #stop() stop} method to start and stop it.
+ *
+ * @author Ludwig Richter (@fussel178)
+ */
+public interface Launcher {
+
+	static Logger logger = LoggerFactory.getLogger(Launcher.class);
```

Comment:
```suggestion
	Logger logger = LoggerFactory.getLogger(Launcher.class);
```

The keyword static is redundant in this case, because it is in an interface.